clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -158,7 +161,8 @@ public NetworkClient(Selectable selector,
              apiVersions,
              null,
              logContext,
-             LeastLoadedNodeAlgorithm.VANILLA);
+             LeastLoadedNodeAlgorithm.VANILLA,
+             Collections.emptyList());
```

Comment:
should we disable cluster metadata refresh future if the bootstrap server is empty?

Reply:
added check